### PR TITLE
Housekeeping tasks

### PR DESCRIPTION
**Pull request by pjvv**
Closes #

Changes proposed in this pull request:

* upgrade ruby
* pin and upgrade gems
* update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,73 +2,81 @@
...
 This should be handled by `bin/setup` above, but if not:
-```
+
     bundle exec rake db:setup
-```
 
 #### (Optional) Create the views
 
-This project provides with the view ```cherrypicked_samples``` that links data with
+This project provides with the view `cherrypicked_samples` that links data with
```
Comment (harrietc52) [inline README.md:43]:
```suggestion
This project provides the view `cherrypicked_samples` that links data with
```

File: README.md [hunk 2]
```diff
@@ -2,73 +2,81 @@
...
-```
+
 ### Integration tests
 
 #### Setup
-1. Initialize the integration tests setup for events warehouse (please check the 
-Integration Tests setup section at <https://github.com/sanger/event_warehouse/>) 
+
+1. Initialize the integration tests setup for events warehouse (please check the
+Integration Tests setup section at <https://github.com/sanger/event_warehouse/>)
```
Comment (harrietc52) [inline README.md:59]:
```suggestion
Integration Tests setup section at <https://github.com/sanger/event_warehouse/#integration-tests-setup>)
```

File: README.md [hunk 3]
```diff
@@ -2,73 +2,81 @@
...
-RAILS_ENV="test"
-INTEGRATION_TEST_SETUP="true"
-```
 
+    RAILS_ENV="test"
+    INTEGRATION_TEST_SETUP="true"
 
 #### Running the integration tests
+
 4. Run the integration tests:
```
Comment (harrietc52) [inline README.md:77]: Don't think this is indented properly as code

**Review by harrietc52 — CHANGES_REQUESTED** [review]
Few v minor comments

File: README.md [hunk 4]
```diff
@@ -2,73 +2,81 @@
...
-```
+
    bundle exec rspec --tag integration
```
Comment (pjvv) [inline README.md:79]:
```suggestion
        bundle exec rspec --tag integration
```

**Pull request merged**